src/LoginPage/LoginForm.jsx

```diff
@@ -1,46 +1,55 @@
 import { memo } from 'react';
 
-export default memo(({ onClick, isLogin }) => ((
-  <form>
-    {isLogin ? null : (
-      <>
-        <input
-          name="first"
-          type="text"
-          placeholder="First Name"
-        />
-        <input
-          name="last"
-          type="text"
-          placeholder="Last Name"
-        />
-      </>
-    )}
-    <input
-      name="email"
-      type="email"
-      placeholder="Email"
-    />
-    <input
-      name="password"
-      type="password"
-      placeholder="Password"
-      autoComplete="off"
-    />
-    {isLogin ? (
-      <button type="submit">Log In</button>)
-      : (<button type="submit">Sign Up</button>)}
+export default memo(({ isLogin, onClick, onChange }) => {
+  function handleChange(event) {
+    const { target: { name, value } } = event;
+    onChange({ name, value });
+  }
 
-    {isLogin ? (<p>Don&apos;t have an account?</p>) : (
-      <p>
-        Passwords must be longer than 7 chars and include numbers
-      </p>
-    )}
-    <button
-      type="button"
-      onClick={onClick}
-    >
-      {isLogin ? 'Sign Up' : 'Log In' }
-    </button>
-  </form>
-)));
+  return (
+    <form>
+      {isLogin ? null : (
```

Comment:
```suggestion
{isLogin && (
```
이런식으로 단축평가 사용하면 동작을 하지 않던가용?~🤔

Reply:
login 과 signup 버튼을 누를 때마다 text 가 바뀌어야 하는 부분들 때문에 삼항 연산자를 사용했는데
알려주신 부분에서는 "논리곱 연산자" 를 사용하는 것이 맞겠네요! 😆